### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56074]
#### VFS team name
Benefits Team 1

#### VFS product name
Supplemental Claims Form 20-0995

#### VFS feature name
Supplemental Claims Form 20-0995

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
If a user uploads evidence and gets an error message, the message reads "something went wrong...". 

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Follow the [files design pattern](https://design.va.gov/patterns/ask-users-for/files#error-state) and change the text to "We couldn't upload your file"

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by Mottie** [comment]
[Message is within the platform code](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/validation.js#L546); but it's a quick update in any case

**Comment by Mottie** [comment]
Fix has been merged and is available in staging for testing

**Comment by saderagsdale** [comment]
@eileen-coforma looks good to me. Good to close?

**Comment by Mottie** [comment]
I don't know... the upload endpoint needs to be down or blocked... turning off the network will show a different message

**Comment by eileen-coforma** [comment]
I can't trigger it, but I trust that it's in the code, so it should be fine.
